Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -174,8 +193,9 @@ def quit_driver_and_reap_children(driver):
         demisto.error(f'Failed checking for zombie processes: {e}. Trace: {traceback.format_exc()}')
 
 
-def rasterize(path: str, width: int, height: int, r_type: str = 'png', wait_time: int = 0,
-              offline_mode: bool = False, max_page_load_time: int = 180, full_screen: bool = False):
+def rasterize(path: str, width: int, height: int, r_type: RasterizeType = RasterizeType.PNG, wait_time: int = 0,
+              offline_mode: bool = False, max_page_load_time: int = 180, full_screen: bool = False,
+              r_mode: RasterizeMode = RasterizeMode.WEBDRIVER_PREFERED):
```

Comment:
Using a global is not recommended. It can cause weird behaviour if the global value is modified. Anyway, here the default is just for completeness of the unit tests. Anyway, the `*_command` functions will pass in the proper mode.